src/components/LuxInputText.vue

```diff
@@ -467,7 +466,6 @@ $color-placeholder: tint(rgb(149, 156, 167), 50%);
     background-color: var(--color-white);
     border-top-right-radius: 3px;
     border-bottom-right-radius: 3px;
-    &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}